orchestration/design.go

```diff
@@ -0,0 +1,51 @@
+package orchestration
+
+import "github.com/meshery/schemas/models/v1beta1/component"
+
+const (
+	ResourceSourceDesignIdLabelKey    = "design.meshery.io/id"
+	ResourceSourceDesignNameLabelKey  = "design.meshery.io/name"
```

Comment:
Perhaps, we can change this to version, as name may change over time, and version might change over time, too.